### DISCUSSION TITLE
ValidateBackHistory.php marked as suspected

### DISCUSSION THREAD

**Discussion by FilippoCalabrese** [area17/twill#879] [General]
Hello,
I'm having problems with the ValidateBackHistory.php middleware which is continuously recognized as a virus by my server. I proceeded to disable the antivirus and restore the file but this is punctually marked as suspect. Is anyone by chance having similar problems?

**Comment by adam-jones-net** [comment]
What program? It's going to be specific to a certain scanner most likely.

**Comment by FilippoCalabrese** [comment]
Hi!
My server uses ClamAV as antivirus but I temporarily disabled it to test this suspicious behavior. Even with the antivirus disabled the problem continues to occur and the file keeps to be renamed in ValidateBackHistory.php.suspected.
